docs/tests.md

```diff
@@ -11,8 +11,13 @@ found in the [general test directory](/test/general).
 ## Defining World Tests
 
 In order to run tests from your world, you will need to create a `test` package within your world package. This can be
-done by creating a `test` directory with a file named `__init__.py` inside it inside your world. By convention, a base
-for your world tests can be created in this file that you can then import into other modules.
+done by creating a `test` directory with an (empty) `__init__.py` inside it inside your world. By convention, a base
```

Comment:
"inside it inside your world" looks like a typo at first, I think this is clearer:
```suggestion
done by creating a `test` directory inside your world with an (empty) `__init__.py` inside it. By convention, a base
```